.github/workflows/manual-appflow-1.yml

```
@@ -68,4 +79,4 @@ jobs:
         with:
           slack-bot-user-oauth-access-token: ${{ secrets.SLACK_BOT_USER_OAUTH_ACCESS_TOKEN }}
           slack-channel: C029QPGHSQL
-          slack-text: ${{github.event.sender.login}} ran manual workflow  on "${{ steps.get-branch.outputs._1 }}" branch \r\n PR link ${{github.event.pull_request.html_url}} \r\n \r\n *Description* \r\n ```${{github.event.pull_request.body}}```  \r\n \r\n APK Link ${{steps.diwaiAndroid.outputs.url}} \r\n IPA link ${{steps.diwaiIos.outputs.url}}
+          slack-text: ${{ github.event.sender.login }} ran manual workflow on "${{ steps.get-branch.outputs._1 }}" branch \r\n PR link ${{ github.event.pull_request.html_url }} \r\n \r\n *Description* \r\n ```${{ github.event.pull_request.body }}```  \r\n \r\n APK Link ${{ needs.build-android.outputs.apkUrl }} \r\n IPA link ${{ needs.build-ios.outputs.iosUrl }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)